### ISSUE TITLE
Idea: show ban date on card box frame

### ISSUE THREAD

**Issue by tooomm** [glacials/whatsinstandard#180]
Same as there is a going to leave date for each box with a group of sets...
![std-msg](https://user-images.githubusercontent.com/9874850/94341409-5c68a680-0009-11eb-93ef-a2bdc899c393.png)

<br>
...there could be the ban date shown:

![ban-msg](https://user-images.githubusercontent.com/9874850/94341406-57a3f280-0009-11eb-931b-0ccd3c63b59e.png)

Related to #181

**Comment by glacials** [comment]
I agree that would be nice, my biggest concern would be whether it would be too wide for the banned card frame. But if we can do it in a way that will hedge against that, I'm down.

**Comment by tooomm** [comment]
Depending on the date style and level of detail (full date vs. week number for example) it could occupy quite some space indeed.

With #181 the frame would get wider as it spans over several cards.
